filebeat/docs/modules-getting-started.asciidoc

```diff
@@ -1,141 +1,238 @@
+----
+PS > .{backslash}{beatname_lc}.exe setup -e
+----
++
+The <<setup-command,`setup`>> command loads the recommended index template for
+writing to {es} and deploys the sample dashboards for visualizing the
+data in {kib}. This is a one-time setup step. 
 +
-If you don't see this message for each log file that needs to be read,
-see <<setting-variables>> to find out how to set the path the files.
+The `-e` flag is optional and sends output to standard error instead of syslog.
 
-NOTE: Depending on how you've installed Filebeat, you might see errors
-related to file ownership or permissions when you try to run Filebeat modules.
+. Run {beatname_uc}:
++
+*deb and rpm:*
++
+["source","sh",subs="attributes"]
+----
+service {beatname_lc} start
+----
++
+*mac:*
++
+["source","sh",subs="attributes"]
+----
+./{beatname_lc} -e
+----
++
+*win:*
++
+["source","sh",subs="attributes"]
+----
+PS > Start-Service {beatname_lc}
+----
++
+If the module is configured correctly, you'll see
+`INFO Harvester started` messages for each file specified in the config.
++
+If your logs aren't in the default location, see <<setting-variables>> to find
+out how to set the path the files, then run {beatname_uc} after you've
+configured the module.
++
+NOTE: Depending on how you've installed {beatname_uc}, you might see errors
+related to file ownership or permissions when you try to run {beatname_uc} modules.
 See {libbeat}/config-file-permissions.html[Config File Ownership and Permissions]
 in the _Beats Platform Reference_ if you encounter errors related to file
-ownership or permissions.
-
-This getting started guide uses the `--modules` flag to enable modules
-at the command line when you run Filebeat. In a production environment, you'll
-probably want to use the configs in the `modules.d` directory instead. See
-<<configuration-filebeat-modules>> for more information.
+ownership or permissions. 
 
 [[setting-variables]]
 ==== Set the path variable
 
 The examples here assume that the logs you're harvesting are in the location
-expected for your OS and that the default behavior of Filebeat is appropriate
+expected for your OS and that the default behavior of {beatname_uc} is appropriate
 for your environment. Each module provides variables that you can set to fine
-tune the behavior of Filebeat, including the location where it looks for log
+tune the behavior of {beatname_uc}, including the location where it looks for log
 files. For example, the following command sets the path for the Nginx access
-logs:
+logs. For this command to work, the `nginx` module must be enabled.
 
-[source,shell]
+// REVIEWERS: I want to provide platform-specific examples, but I'm not sure
+// what theses commands look like for deb/rpm/windows. I assume we want to
+// tell users to start the service, but I don't think options like -M are
+// allowed when you start as a service. Are they? Hoping someone with
+// deb/rpm/windows experience can help me get the syntax right here.
```

Comment:
There is no way to pass these variables to services, they would need to update the config file with them

Reply:
@exekias Then I should probably add a note to indicate that. But I think users can still run the following commands, right?

deb and rpm:
```
filebeat -e -M "nginx.access.var.paths=[/var/log/nginx/access.log*]"
```

win:
```
PS > .\filebeat.exe -e -M "nginx.access.var.paths=[c:/programdata/nginx/logs/*access.log*]"
```

Reply:
Those look correct :+1: but they are not useful to keep the service running, but for testing.